databook/physiology/ephys/np-ultra/npultra-psychedelics.md

```diff
@@ -0,0 +1,58 @@
+
+:::{figure} ../resources/NPUltra/Jakab_etal_1998_5HT2AR_localization.png
+:name: 5HT2AR cellular localization
+:align: center
+:width: 800
+
+Localization of the $5HT_{2A}R$ in the primate cortex. __A__, Dark-field photographs show the distribution of 5-HT2A receptor immunoreactivity in two representative sections of the prefron- tal cortex of an adult macaque monkey. __B__, Inset from __A__ showing $5HT_{2A}R$ labeling distribution in the cortical column. __C__ & __D__, insets from __B__. Arrows in __C__ indicate the strongest immunoreactivity in the apical dendrites of L5p neurons. From Jakab & Goldman-Rakic (1998)
+:::
+
+Current hypotheses of psychedelic effects on the nervous system suggest that the perceptual and cognitive effects of these drugs are related to alterations in pyramdial neuron dendritic excitability via $5HT_{2A}R$ activation {cite:p}`Kwan2022`. Though the exact acute and long-term effects of $5HT_{2A}R$ activation on network activity have yet to be fully elucidated, a growing concensus points toward psychedelic drugs altering dendritic integration and shifting the spike-time distribution of pyramdial neurons during the course of drug exposure while promoting long-term synaptic plasticity and potentiation (LTP). Determining the precise cell-type-specific effects on neuronal firing of psychedelic drug exposure represents and important next step for understand both the therapeutic mechanisms of these compounds as well as what they can tell us about serotonergic signaling in the cortex
+
+:::{figure} ../resources/NPUltra/Kwan_etal_2022_NeuralBasis.png
+:name: 5HT2A mechanism of action
+:align: center
+:width: 800
+
+$5HT_{2A}R$ mechanism of action. __A__, Intracellular signal transduction pathways. Downstream of the 5-HT2A receptor, activation of heterotrimeric G proteins and subsequent intracellular signaling (Ca2+ release and diacylglycerol (DAG) production) synergistically activate additional downstream effects, which ultimately lead to altered neuronal firing; PIP2, phosphatidylinositol-4,5-bisphosphate; PLC-β, phospholipase C-β; ER, endoplasmic reticulum; IP3, inositol trisphosphate. __B__, *Left*, structure of the 5-HT2A receptor; right, model of the 5-HT2A receptor signaling complex in the membrane.
+:::
+
+## Dataset
+
+The Neuropixels Ultra & Psychedelics dataset was collected to explore how different subtypes of L5p respond to psilocybin during passive spontaneous activity and with visual stimulation. The two subtypes of L5p identified and assessed in these experiments are defined by the transgenes Sim1-Cre;Ai32 and Tlx3-Cre;Ai167 and correspond to thick-tufted extratelencephalic (ET)-projecting and thin-tufted intratelencephalic (IT)-projecting neurons, respectively.
```

Comment:
have we added these to the transgenic tools page? Also, are these the real Cre line names? Eg. we use Tlx3-Cre_PL56 for the vis coding project.

You need to define ET and IT. I know you have the word, but that's not a word that people know. Either add it to the glossary and define it there, or unpack it here.

Reply:
Ai167 needs to be added to the transgenic tool page too.